apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/PolicyDetailsV2.tsx

```diff
@@ -30,6 +31,8 @@ interface PolicyDetailsV2Props {
 export const PolicyDetailsV2 = ({ isEditing, form, onUpdateCommand }: PolicyDetailsV2Props) => {
   const { project } = useProjectContext()
   const snap = useTableEditorStateSnapshot()
+  const router = useRouter()
+  const searchTableId = router.query.search as string
```

Comment:
`const { search } = useParams()`

Reply:
also - realised this will be an odd behaviour if i cleared the UI search input, but it still defaults to the table in the URL search param. will update